tests/pass/concurrency/sync.rs

```diff
@@ -1,4 +1,5 @@
 // ignore-windows: Concurrency on Windows is not supported yet.
+// ignore-android: `libc` crate does not support `gettimeofday()` on Android
```

Comment:
Please reword this to document which stdlib API Miri does not support on Android. (I assume with rustc, this code here works fine on Android.)